openmetadata-ui/src/main/resources/ui/src/components/EntityTable/EntityTable.component.tsx

```diff
@@ -680,20 +680,22 @@ const EntityTable = ({
         data-testid="entity-table"
         dataSource={data}
         expandable={{
-          defaultExpandedRowKeys: [],
-          expandIcon: ({ expanded, onExpand, record }) =>
-            record.children ? (
-              <FontAwesomeIcon
-                className="tw-mr-2 tw-cursor-pointer"
-                icon={expanded ? faCaretDown : faCaretRight}
+          rowExpandable: (record) => {
+            return (record.children && record.children.length > 0) || false;
+          },
```

Comment:
I would recommend this way to check if the row is expandable or not.

```suggestion
          rowExpandable: (record) => record.children && record.children.length > 0,
```

Reply:
Thanks, @Sachin-chaurasiya but we need to pass a boolean value, so if children are undefined it will send undefined and its not accepted in function so we need to give fallback value
<img width="1010" alt="image" src="https://user-images.githubusercontent.com/71748675/203044107-86f6e280-949d-4c3f-b921-5f4a8e85e249.png">

Reply:
Okay, then can we do this?
```suggestion
          rowExpandable: (record) => Boolean(record.children && record.children.length > 0),
```